manuscript/algorithm.tex

```diff
@@ -0,0 +1,77 @@
+\section{ Demand-driven deployment models}
+To predict the market demand for nuclear fuel cycle simulators, various algorithms
+can be adopted. The algorithms can be divided into three major
+branches, Non-optimizing, Deterministic Optimizing, and Stochastic
+Optimizing. Each groups vary in complexity, which leads to a difference
+in computing time and accuracy. The groups and their examples are 
+explained in detail below. 
+
+\subsection{Non-Optimizing(NO)}
+
+Non optimizing algorithms predict future deployment schedules
+only based on the past datasets. These algorithms do not attempt
+to minimize the difference of the produced quantity to the demand,
+thus the name non-optimizing. The simple nature of the algorithm
+allows fast execution time, but a drawback of low precision. The
+two main algorithms explored are \gls{ARMA}
+and \gls{ARCH} methods. 
+
+\subsubsection{Autoregressive Moving Average (ARMA)}
+
+\gls{ARMA} is a combination of two methods, the Autoregressive
+and the Moving Average model. The Autoregressive model 
+predicts future values by fitting a linear curve fit of the latest
+datasets, and the Moving Average method does so by fitting the 
+errors. \cite{intro_arma}
+
+The model is referred to as ARMA(p,q) format, where the p and q represent 
+the order of the autoregressive, and the moving average part,
```

Comment:
the specific meaning of order in this context should be explained here.
part-> parts